tests/MassTransit.Azure.Table.Tests/AzureTableInMemoryTestFixture.cs

```diff
@@ -0,0 +1,57 @@
+namespace MassTransit.Azure.Table.Tests
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using Microsoft.Azure.Cosmos.Table;
+    using NUnit.Framework;
+    using TestFramework;
+
+
+    public class AzureTableInMemoryTestFixture :
+        InMemoryTestFixture
+    {
+        protected readonly string AccessKey;
+        protected readonly string AccountName;
+        protected readonly string ConnectionString;
+        protected readonly string TableEndpoint;
+        protected readonly CloudTable TestCloudTable;
+        protected readonly string TestTableName;
+
+        public AzureTableInMemoryTestFixture()
+        {
+            AccountName = "localhost";
```

Comment:
Does this work with the Cosmos Emulator? Curious how this can be tested via CI. Also, consider using the Configuration.cs approach used by Azure Service Bus Tests so that environment variables can be used for test connection settings.

Reply:
I've switched it to Storage account and using the standard configuration approach from ASB. I've added a local config in for a CosmosDB Emulator purely for convenience if everything is null